packages/salesforce-adapter/test/filters/field_references.test.ts

```diff
@@ -78,6 +79,18 @@ import { FilterWith } from './mocks'
 import { buildFetchProfile } from '../../src/fetch_profile/fetch_profile'
 
 const { awu } = collections.asynciterable
+const { isDefined } = values
+
+const getDefKey = (def: FieldReferenceDefinition): string => {
+  const parts = [
+    def.src.parentTypes.join('_'),
+    def.src.field,
+    def.target?.type,
+    def.target?.parentContext,
+    def.target?.typeContext,
+  ].filter(isDefined)
+  return parts.join('@')
```

Comment:
I'd rather go with something a bit more readable here. Maybe:
```suggestion
  const srcParts = [
    def.src.parentTypes.join('_'),
    def.src.field,
  ].filter(isDefined)
  const targetParts = [
    def.target?.type,
    def.target?.parentContext,
    def.target?.typeContext,
  ].filter(isDefined)
  return `${srcParts.join('.')}:${targetParts.join('.')}`
```
This will give us (to take what looks like one of the worst examples):
```
FlowAssignmentItem_FlowStageStepOutputParameter_FlowSubflowOutputAssignment_FlowTransformValueAction_FlowScreenFieldOutputParameter_FlowWaitEventOutputParameter_FlowStageStepExitActionOutputParameter_FlowApexPluginCallOutputParameter_FlowActionCallOutputParameter_FlowOutputFieldAssignment_FlowStageStepEntryActionOutputParameter.assignToReference:CustomField.instanceParent
```
Which IMO is a bit cleaner (and in simpler cases I think it's much cleaner).